src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -69,6 +69,7 @@
   "enrollmentVerification": "enrollment_verification",
   "ezrProdEnabled": "ezr_prod_enabled",
   "ezrUploadEnabled": "ezr_upload_enabled",
+  "ezrAuthOnlyEnabled": "ezr_auth_only_enabled",
```

Comment:
Ideally, this should be its own PR ahead of any body of work, to avoid having to get platform involved in checking your full PR. Please note this for future work.